articles/azure-monitor/app/custom-operations-tracking.md

```diff
@@ -364,8 +348,15 @@ When you instrument message deletion, make sure you set the operation (correlati
 - Stop the `Activity`.
 - Use `Start/StopOperation`, or call `Track` telemetry manually.
 
+### Dependency Types
+
+Application Insights uses dependency type to cusomize UI experiences. For queues it recognizes following types of `DependencyTelemetry` that improve [Transaction diagnostics experience](../../azure-monitor/app/transaction-diagnostics):
+- `Azure queue` for Azure Storage Queues
+- `Azure Event Hubs` for Azure Event Hubs
+- `Azure Service Bus` for Azure Service Bus
+
 ### Batch processing
-With some queues, you can dequeue multiple messages with one request. Processing such messages is presumably independent and belongs to the different logical operations. In this case, it's not possible to correlate the `Dequeue` operation to particular message processing.
+With some queues, you can dequeue multiple messages with one request. Processing such messages is presumably independent and belongs to the different logical operations. It's not possible to correlate the `Dequeue` operation to particular message processing.
```

Comment:
I realize the original had the same sentence structure, but on reread I think something like the below might flow a little bit better. (If this is losing precision just let me know and I will come up with an alternate suggestion.)

```suggestion
With some queues, you can dequeue multiple messages with one request. Processing such messages is presumably independent and belongs to the different logical operations. It's not possible to correlate the `Dequeue` operation to a particular message being processed.
```
or